src/trinerdi/sc.sh

```diff
@@ -1,9 +1,11 @@
 e(){ # run on [problem].in*
```

Comment:
What does this do now? Maybe it would be useful to have a comment describing what the different usage possibilities are

Reply:
You're right, I've put a comment above that function. I've also changed the default test input wildcard from `[problem]*in*` to just `*in*`.